src/Resources/app/storefront/src/js/enableSubmitAfterPageLoad.js

```diff
@@ -1,4 +1,4 @@
-window.onload = function(){
+window.addEventListener('load', (event) => {
```

Comment:
Minor: JavaScript does not check the function's signature like normal languages do ;-), and it's usually considered a good practice (well, at least according to the PhpStorm's linter, I think) to omit parameters that are not actually used.